modules/nw-operator-cr.adoc

```diff
@@ -265,10 +265,10 @@ endif::operator[]
 |`object`
 |
 ifndef::operator[]
-Specify an empty object to enable IPsec encryption.
+Specify a configuration object for customizing the IPsec configuration.
 endif::operator[]
 ifdef::operator[]
-If the field is present, IPsec is enabled for the cluster.
+If an object is specified, IPsec is enabled for the cluster.
```

Comment:
This is inaccurate - an object can be specified also when the mode is `Disabled`. Ipsec is enabled only with modes `Full` or `External`

Reply:
@JoshSalomon, if a user does nothing during cluster installation to change this, the default value is `Disabled`?

Reply:
@jab-rh Yes